app/src/main/java/co/tinode/tindroid/db/MessageDb.java

```diff
@@ -301,6 +301,53 @@ static Cursor queryDeleted(SQLiteDatabase db, long topicId, boolean hard) {
         return db.rawQuery(sql, null);
     }
 
+    /**
+     * Find the latest missing range of messages for fetching from the server.
```

Comment:
Imagine there are no deleted messages in given topic.
Then, say, we've got message seqs 1, 2, 3 and 7, 5, 9 in Messages table.
This function should return [4, 7), correct?

Reply:
Correct.